src/StorageWriter/DatabaseMongo.ts

```diff
@@ -0,0 +1,66 @@
+import { Claim } from '@po.et/poet-js'
```

Comment:
Weren't we calling these WhateverDAO to avoid conflicting with Mongo's Database? (Just noticed BatchWriter's FileDAO)

Reply:
We were calling them by that format, but not because of conflicts with Mongo's naming. The reason was that each DAO was dealing with a single collection. But that means that the claim controller needs to know about both collections independently. I thought it might be better for it to just deal with a single database interface, but I could go either way on this 🤷‍♂️

Reply:
Yep, one DAO per collection sounds best, but at the level of complexity we're at it doesn't really make any difference. 

Either way I'm fine, but I do feel strongly about not calling them "Database" or "Mongo" or "DatabaseMongo". I'd like to continue using DAO, mostly because it is (or used to be) the standard, but def open to suggestions.